File: check_merchants.py
import psycopg2
conn = psycopg2.connect(host='127.0.0.1', port=5432, dbname='prix_tunisix', user='postgres', password='050114')
cur = conn.cursor()
cur.execute("SELECT id, name, base_url FROM merchant_websites WHERE name ILIKE '%tunisia%'")
print([r for r in cur.fetchall()])
conn.close()

File: backend/check_merchants.php
<?php
$pdo = new PDO('pgsql:host=127.0.0.1;port=5432;dbname=prix_tunisix', 'postgres', '050114');
$merchants = $pdo->query("SELECT id, name FROM merchant_websites ORDER BY id")->fetchAll(PDO::FETCH_ASSOC);
foreach ($merchants as $m) {
    $name = strtolower(str_replace([' ', '-', '_'], '', $m['name']));
    echo "{$m['id']}: {$m['name']} -> normalized: $name\n";
}